tests/integration/transformers/prefix.phpt

```diff
@@ -0,0 +1,84 @@
+--FILE--
+<?php
+
+declare(strict_types=1);
+
+require 'vendor/autoload.php';
+
+use Respect\Validation\Validator as v;
+
+run([
+    'key' => [v::keyEquals('foo', 12), ['foo' => 10]],
+    'length' => [v::lengthGreaterThan(3), 'foo'],
+    'max' => [v::maxOdd(), [1, 2, 3, 4]],
+    'min' => [v::minEven(), [1, 2, 3]],
+    'not' => [v::notBetween(1, 3), 2],
+    'nullOr' => [v::nullOrBoolType(), 'string'],
+    'property' => [v::propertyBetween('foo', 1, 3), (object) ['foo' => 5]],
+    'undefOr' => [v::undefOrUrl(), 'string'],
+]);
+?>
+--EXPECT--
+key
+⎺⎺⎺
+foo must equal 12
+- foo must equal 12
+[
+    'foo' => 'foo must equal 12',
+]
+
+length
+⎺⎺⎺⎺⎺⎺
+The length of "foo" must be greater than 3
+- The length of "foo" must be greater than 3
+[
+    'length' => 'The length of "foo" must be greater than 3',
```

Comment:
This is an example of something that could be tricky. The actual rule is "length" but what the user called was "lengthGreaterThan"